R/server.R

```diff
@@ -590,6 +600,156 @@ shinyServer(function(input, output, session) {
         removeModal()
     }, ignoreInit = TRUE)
     
+    
+    #### -------------------------- SLIP DAYS ----------------------------####
+
+    slip_days <- reactiveValues(
+        table = list(),
+        edit = NULL
+    )
+    
+    # Opening category modal to create a NEW SLIP DAYS POLICY
+    observeEvent(input$new_slip_days, {
+        showModal(edit_slip_days_modal)
+        current_edit$slip_days <- NULL
+        updateTextInput(session, "slip_days_policy_name", value = "Slip Days Policy 1")
+        updateNumericInput(session, "num_slip_days", value = 0)
+        updateSelectInput(session, "slip_days_order", selected = "chronological")
+        if (!is.null(assign$table)) {
+            updateSelectizeInput(session, "assignments", choices = assign$table$assignment, selected = NULL)
```

Comment:
Let's try setting choices = c(all assignments, all categories)... I don't think it should break the app _and_ I'll add some recursive function that replaces all categories with a list of their assignments

Reply:
will do this after end-of-semester demo.